.nuget/NuGet.targets

```diff
@@ -20,8 +20,8 @@
         <!-- Package sources used to restore packages. By default, registered sources under %APPDATA%\NuGet\NuGet.Config will be used -->
         <!-- The official NuGet package source (https://nuget.org/api/v2/) will be excluded if package sources are specified and it does not appear in the list -->
         
-            <PackageSource Include="https://nuget.org/api/v2/" />
-            <PackageSource Include="http://nupeek.internal.azavea.com" />
+            <PackageSource Include="http://nuget.org/api/v2/" />
```

Comment:
You, get the following error with `https` on Jenkins/LR01 when fetching packages:

```
error : The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. 
```